src/test/java/com/powsybl/openloadflow/ac/AcLoadFlowEurostagTutorialExample1Test.java

```diff
@@ -409,4 +409,26 @@ void testWithDisconnectedGenerator() {
         assertActivePowerEquals(Double.NaN, gen.getTerminal());
         assertReactivePowerEquals(Double.NaN, gen.getTerminal());
     }
+
+    @Test
+    void testGeneratorReactiveLimits() {
```

Comment:
Actually, that test also passed without the evolution --> we should try to adapt it so that it really tests the fix.